src/Shikensu/Utilities.hs

```diff
@@ -52,10 +53,10 @@ lsequence list =
             List.unzip list
 
         identifiers =
-            Tuple.fst unzippedList
+            fst unzippedList
```

Comment:
I know, but I like to prefix it with `Tuple` so I know, at all times, that I'm working with a tuple. Same goes for lists.